### ISSUE TITLE
Is this dead?

### ISSUE THREAD

**Issue by tiagozip** [ai/evil-blocks#43]
Welp, it's 2024 lol

**Comment by ai** [comment]
Yes, there is no reason to use it today and I do not plan to add new features

**Comment by tiagozip** [comment]
oh ok. I guess I'll just use svelte, it can do the same thing lol

Andrey Sitnik ***@***.***> escreveu (quarta, 27/03/2024 à(s)
15:30):
